### PR TITLE
Add Eclipse60 / tinyneko

### PR DESCRIPTION
**Pull request by MaiTheSan**
Added Eclipse60 / tinyneko support

## Description
Add Eclipse60 / tinyneko 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sawnsprojects/eclipse/eclipse60/info.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    "rgblight": {
+        "pin": "B0",
+        "led_count": 18,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
+        "sleep": true,
+        "animations": {
+            "all": true
```
Comment (waffle87) [inline keyboards/sawnsprojects/eclipse/eclipse60/info.json:39]:
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

File: keyboards/sawnsprojects/eclipse/tinyneko/info.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    "rgblight": {
+        "pin": "B0",
+        "led_count": 18,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
+        "sleep": true,
+        "animations": {
+            "all": true
```
Comment (waffle87) [inline keyboards/sawnsprojects/eclipse/tinyneko/info.json:39]:
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

File: keyboards/sawnsprojects/eclipse/eclipse60/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# eclipse60
+
+![eclipse60](i.imgur.com/0nttqlt.png)
```
Comment (waffle87) [inline keyboards/sawnsprojects/eclipse/eclipse60/readme.md:3]:
```suggestion
![eclipse60](i.imgur.com/0nttqlth.png)
```

File: keyboards/sawnsprojects/eclipse/tinyneko/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# tinyneko
+
+![tinyneko](https://i.imgur.com/hs8RWzL.png)
```
Comment (waffle87) [inline keyboards/sawnsprojects/eclipse/tinyneko/readme.md:3]:
```suggestion
![tinyneko](https://i.imgur.com/hs8RWzLh.png)
```

File: keyboards/sawnsprojects/eclipse/eclipse60/info.json [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        }
+    },
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (waffle87) [inline keyboards/sawnsprojects/eclipse/eclipse60/info.json:43]: `LAYOUT_all` is generally used for boards that support multiple layouts. `LAYOUT` should be used here instead. Same goes for `keyboards/sawnsprojects/eclipse/tinyneko`.

**Review by waffle87 — CHANGES_REQUESTED** [review]
`keyboards/sawnsprojects/eclipse/eclipse60/rules.mk` & `keyboards/sawnsprojects/eclipse/tinyneko/rules.mk` need to match [data/templates/keyboard/rules.mk](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/rules.mk)

**Review by drashna — APPROVED** [review]

File: keyboards/sawnsprojects/eclipse/eclipse60/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# eclipse60
+
+![eclipse60](i.imgur.com/0nttqlth.png)
```
Comment (fauxpark) [inline keyboards/sawnsprojects/eclipse/eclipse60/readme.md:3]:
```suggestion
![eclipse60](https://i.imgur.com/0nttqlth.png)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by MaiTheSan** [comment]
Thanks~

**Pull request merged**